openmm_septop/python_tutorial.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<p>"describes all the particles in the simulation" - same comment as for ABFEs, I know this is just re-using previous text (which we need to change with something a bit less confusing for users).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>